iocuddle-sgx/Cargo.toml

```diff
@@ -13,5 +13,5 @@ sgx-types = { path = "../sgx-types" }
 
 [dev-dependencies]
 sgx-crypto = { path = "../sgx-crypto" }
-openssl = "0.10.26"
+openssl = { version = "0.10.26", features = [ "vendored" ] }
```

Comment:
I'd really like to avoid this if at all possible. Long term, we'd love to use the `openssl-static` build from RHEL. What stands in our way from doing that today?

Reply:
We can't link the static libs generated by gcc with glibc include headers with musl libc. So, the openssl-static on fedora doesn't help here. It would have to be recompiled with musl-gcc.

Reply:
Understood. So we have a longer term goal of static linking on glibc.

Reply:
> Understood. So we have a longer term goal of static linking on glibc.

see https://github.com/enarx/enarx/issues/6#issuecomment-578021892

Reply:
Are we able to inject the vendored option only when building inside our workspace? I'm asking because it is possible for someone to use one of our crates as a dependency in a normal context. I don't want to force vendored on them.

Reply:
I did some research on this and I think the answer is no. :(